File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/ActiveSaleDeliveriesRetailer.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class ActiveSaleDeliveriesRetailer {
    @JsonProperty("AcceptedDateTime")
    public String acceptedDateTime;
    @JsonProperty("ActualDepartureDateTime")
    public String actualDepartureDateTime;
    @JsonProperty("AllowFullEdit")
    public Boolean allowFullEdit;
    @JsonProperty("CompletedDateTime")
    public String completedDateTime;
    @JsonProperty("DateTime")
    public String dateTime;
    @JsonProperty("Destinations")
    public List<Object> destinations;
    @JsonProperty("Direction")
    public String direction;
    @JsonProperty("DriverEmployeeId")
    public String driverEmployeeId;
    @JsonProperty("DriverName")
    public String driverName;
    @JsonProperty("DriversLicenseNumber")
    public String driversLicenseNumber;
    @JsonProperty("EstimatedDepartureDateTime")
    public String estimatedDepartureDateTime;
    @JsonProperty("FacilityLicenseNumber")
    public String facilityLicenseNumber;
    @JsonProperty("FacilityName")
    public String facilityName;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("LastModified")
    public String lastModified;
    @JsonProperty("Leg")
    public Integer leg;
    @JsonProperty("Packages")
    public List<Object> packages;
    @JsonProperty("RecordedByUserName")
    public String recordedByUserName;
    @JsonProperty("RecordedDateTime")
    public String recordedDateTime;
    @JsonProperty("RestockDateTime")
    public String restockDateTime;
    @JsonProperty("RetailerDeliveryNumber")
    public String retailerDeliveryNumber;
    @JsonProperty("RetailerDeliveryState")
    public String retailerDeliveryState;
    @JsonProperty("TotalPackages")
    public Integer totalPackages;
    @JsonProperty("TotalPrice")
    public Integer totalPrice;
    @JsonProperty("TotalPriceSold")
    public Integer totalPriceSold;
    @JsonProperty("VehicleInfo")
    public String vehicleInfo;
    @JsonProperty("VehicleLicensePlateNumber")
    public String vehicleLicensePlateNumber;
    @JsonProperty("VehicleMake")
    public String vehicleMake;
    @JsonProperty("VehicleModel")
    public String vehicleModel;
    @JsonProperty("VoidedDate")
    public String voidedDate;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/ActiveSaleDeliveriesRetailer.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class ActiveSaleDeliveriesRetailer(
    @JsonProperty("AcceptedDateTime")
    val acceptedDateTime: String? = null,
    @JsonProperty("ActualDepartureDateTime")
    val actualDepartureDateTime: String? = null,
    @JsonProperty("AllowFullEdit")
    val allowFullEdit: Boolean? = null,
    @JsonProperty("CompletedDateTime")
    val completedDateTime: String? = null,
    @JsonProperty("DateTime")
    val dateTime: String? = null,
    @JsonProperty("Destinations")
    val destinations: List<Any>? = null,
    @JsonProperty("Direction")
    val direction: String? = null,
    @JsonProperty("DriverEmployeeId")
    val driverEmployeeId: String? = null,
    @JsonProperty("DriverName")
    val driverName: String? = null,
    @JsonProperty("DriversLicenseNumber")
    val driversLicenseNumber: String? = null,
    @JsonProperty("EstimatedDepartureDateTime")
    val estimatedDepartureDateTime: String? = null,
    @JsonProperty("FacilityLicenseNumber")
    val facilityLicenseNumber: String? = null,
    @JsonProperty("FacilityName")
    val facilityName: String? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("LastModified")
    val lastModified: String? = null,
    @JsonProperty("Leg")
    val leg: Int? = null,
    @JsonProperty("Packages")
    val packages: List<Any>? = null,
    @JsonProperty("RecordedByUserName")
    val recordedByUserName: String? = null,
    @JsonProperty("RecordedDateTime")
    val recordedDateTime: String? = null,
    @JsonProperty("RestockDateTime")
    val restockDateTime: String? = null,
    @JsonProperty("RetailerDeliveryNumber")
    val retailerDeliveryNumber: String? = null,
    @JsonProperty("RetailerDeliveryState")
    val retailerDeliveryState: String? = null,
    @JsonProperty("TotalPackages")
    val totalPackages: Int? = null,
    @JsonProperty("TotalPrice")
    val totalPrice: Int? = null,
    @JsonProperty("TotalPriceSold")
    val totalPriceSold: Int? = null,
    @JsonProperty("VehicleInfo")
    val vehicleInfo: String? = null,
    @JsonProperty("VehicleLicensePlateNumber")
    val vehicleLicensePlateNumber: String? = null,
    @JsonProperty("VehicleMake")
    val vehicleMake: String? = null,
    @JsonProperty("VehicleModel")
    val vehicleModel: String? = null,
    @JsonProperty("VoidedDate")
    val voidedDate: String? = null
)
